pkg/stub/handler_test.go

```diff
@@ -555,7 +555,7 @@ func validate(succeed bool, err error, errstr string, sr *v1alpha1.SamplesResour
 			case v1alpha1.ImportCredentialsExist:
 				fallthrough
 
-			case v1alpha1.SamplesExist:
+			case v1alpha1.SamplesExist, v1alpha1.ConfigurationValid:
```

Comment:
you can bring the fallthrough case above down to this line too.

Reply:
yep part of next push